File: examples/java/kora-java-crud-submodule/kora-java-crud-submodule-vet-api/src/main/java/io/koraframework/example/submodule/vet/service/VetCache.java
package io.koraframework.example.submodule.vet.service;

import io.koraframework.cache.annotation.Cache;
import io.koraframework.cache.caffeine.CaffeineCache;
import io.koraframework.example.submodule.vet.model.dao.Vet;

@Cache("vet-cache")
public interface VetCache extends CaffeineCache<Long, Vet> {

}

File: examples/kotlin/kora-kotlin-crud-submodule/kora-kotlin-crud-submodule-vet-api/src/main/kotlin/io/koraframework/kotlin/example/submodule/vet/service/VetCache.kt
package io.koraframework.kotlin.example.submodule.vet.service

import io.koraframework.cache.annotation.Cache
import io.koraframework.cache.caffeine.CaffeineCache
import io.koraframework.kotlin.example.submodule.vet.model.dao.Vet

@Cache("vet-cache")
interface VetCache : CaffeineCache<Long, Vet>
